RNTester/js/RNTesterExampleContainer.js

```diff
@@ -32,17 +44,81 @@ class RNTesterExampleContainer extends React.Component {
     );
   }
 
+  renderExamples() {
+    const {module} = this.props;
+    const {filterText} = this.state;
+    let filterRegex = /.*/;
+
+    try {
+      filterRegex = new RegExp(String(filterText), 'i');
+    } catch (error) {
+      console.warn(
+        'Failed to create RegExp: %s\n%s',
+        filterText,
+        error.message,
+      );
+    }
+
+    const filter = example => filterRegex.test(example.title);
+
+    return module.examples.filter(filter).map(this.renderExample);
+  }
+
+  renderFilter() {
+    const {filterText} = this.state;
+
+    return (
+      <View style={styles.searchRow}>
+        <TextInput
+          autoCapitalize="none"
+          autoCorrect={false}
+          clearButtonMode="always"
+          onChangeText={text => {
+            this.setState(() => ({filterText: text}));
+          }}
+          placeholder="Search..."
+          underlineColorAndroid="transparent"
+          style={styles.searchTextInput}
+          testID="explorer_example_search"
+          value={filterText}
+        />
+      </View>
+    );
+  }
+
   render(): React.Element<any> {
     if (!this.props.module.examples) {
       return <this.props.module />;
     }
 
+    if (this.props.module.examples.length === 1) {
```

Comment:
Here are the main changes in this pull

If there is only one example exported, we just show the single example. This should maintain the existing behavior for modules until they're refactored to export multiple examples.

If there's more than one example exported, we add a filter which will filter by the example title